packages/admin-ui/src/common/routes.ts

```diff
@@ -105,6 +105,10 @@ export interface Routes {
     toPath: string;
   }) => Promise<void>;
 
+  /** Set the dappnodeWebNameSet */
+
+  dappnodeWebNameSet: (kwargs: { dappnodeWebName: string }) => Promise<void>;
```

Comment:
You can simplify the types by doing
```ts
dappnodeWebNameSet: (dappnodeWebName: string) => Promise<void>;
```
I think there's no room for confusion here